apps/dashboard/app/models/user_configuration.rb

```diff
@@ -54,6 +57,11 @@ class UserConfiguration
     ConfigurationProperty.property(name: :custom_css_files, default_value: []),
 
     ConfigurationProperty.property(name: :dashboard_title, default_value: 'Open OnDemand', read_from_env: true),
+
+    # Navigation properties
+    ConfigurationProperty.with_boolean_mapper(name: :show_all_apps_link, default_value: false, read_from_env: true, env_names: ['SHOW_ALL_APPS_LINK']),
+    ConfigurationProperty.with_boolean_mapper(name: :navconfig_categories_filter, default_value: NavConfig.categories_whitelist?),
+    ConfigurationProperty.property(name: :navconfig_categories, default_value: NavConfig.categories),
```

Comment:
I think this for sure can be `nav_categories` or `nav_bar_categories`. "Nav bar" may be a bit esoteric for regular folks, but docs could help? I don't think we can use `top_bar_categories`, I'm like 90% sure we have to keep `nav` or `navigation`.

In any case the `config` is implied so we don't really need it.

Reply:
I have updated both to `nav_categories_filter` and `nav_categories` to see how they look.